acl_upgrade.php

```diff
@@ -728,23 +728,33 @@
     $acl_version = $upgrade_acl;
 }
 
-/* This is a template for a new revision, when needed
 // Upgrade for acl_version 10
 $upgrade_acl = 10;
 if ($acl_version < $upgrade_acl) {
-    echo "<B>UPGRADING ACCESS CONTROLS TO VERSION ".$upgrade_acl.":</B></BR>";
+    echo "<B>UPGRADING ACCESS CONTROLS TO VERSION " . $upgrade_acl . ":</B></BR>";
 
     //Collect the ACL ID numbers.
     echo "<B>Checking to ensure all the proper ACL(access control list) are present:</B></BR>";
 
     //Add new object Sections
     echo "<BR/><B>Adding new object sections</B><BR/>";
+    AclExtended::addObjectSectionAcl('inventory', 'Inventory');
 
     //Add new Objects
     echo "<BR/><B>Adding new objects</B><BR/>";
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'lots', 'Lots');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'sales', 'Sales');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'purchases', 'Purchases');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'transfers', 'Transfers');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'adjustments', 'Adjustments');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'consumption', 'Consumption');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'destruction', 'Destruction');
+    AclExtended::addObjectAcl('inventory', 'Inventory', 'reporting', 'Reporting');
 
     //Update already existing Objects
     echo "<BR/><B>Upgrading objects</B><BR/>";
+    //Rename "Pharmacy Dispensary" to "Inventory Administration".
+    AclExtended::editObjectAcl('admin', 'Administration', 'drugs', 'Inventory Administration', 10);
 
     //Add new ACLs here (will return the ACL ID of newly created or already existant ACL)
```

Comment:
Do these new acos need to be added to an acl?

Reply:
Note superuser (admin/super) gets access to everything now. My thinking was that instead of stepping on the site's existing groups, let the site admin decide which groups (roles) get the different inventory ACOs. But am open to suggestions.

Reply:
Rec placing all acos in the Administration and Emergency User ACLs (this has been standard practice and ensures no acos get left behind :) )

Reply:
Done.